content/influxdb/v2.2/write-data/replication.md

```diff
@@ -14,7 +14,7 @@ related:
   - /influxdb/v2.2/reference/cli/influx/replication
 ---
 
-Use InfluxDB replication streams to replicate all data written to an InfluxDB OSS
+Use InfluxDB Edge Data Replication to replicate all data written to an InfluxDB OSS
```

Comment:
```suggestion
Use InfluxDB Edge Data Replication to replicate the incoming data of select buckets to one or more buckets on a remote InfluxDB instance.
```

Reply:
thanks, @samhld, re including all the "targets" via the v2 api.... the way I read this doc heading "Replicate data from InfluxDB OSS to InfluxDB Cloud" is targeting a Cloud bucket specifically. I agree we should document all the ways moving forward that are supported. lmk if I'm missing something about your suggestion here.